src/components/Voting/VotingCard.tsx

```diff
@@ -4,15 +4,31 @@ import RemoveMarkdown from 'remove-markdown'
 import Card from 'components/Card'
 import FormattedDate from 'components/Voting/FormattedDate'
 
-import { VotingProposalProps } from 'models/voting'
+import {
+  VotingProposalProps,
+  VotingResultStatus,
+  VotingStatus,
+} from 'models/voting'
 
 interface VotingCardProps {
   proposal: VotingProposalProps
 }
 
 const VotingCard: React.FC<VotingCardProps> = ({
-  proposal: { state, title, body, start, id },
+  proposal: { state, title, body, start, id, choices, scores },
 }) => {
+  let stateText = state
+  if (state === VotingStatus.CLOSED) {
+    const highestVotedIndex = scores?.indexOf(Math.max(...scores))
+    if (choices[highestVotedIndex] === 'For') {
+      stateText = VotingResultStatus.PASSED
+    } else if (choices[highestVotedIndex] === 'Against') {
```

Comment:
This warning is showing for all if else logics here maybe because `choices[highestVotedIndex]` needs to be declared as a string. Could you try `choices[highestVotedIndex].toString()`?
![image](https://user-images.githubusercontent.com/26428605/183448366-4d9aab9f-547a-43c3-b9fb-a9c089c03283.png)